### PR TITLE
Add prompt template from langfuse

### PR DESCRIPTION
**Pull request by cruz-lucas**
I added a prompt component that fetches a prompt from LangFuse's repository.

The credentials come from the environment variables, that is, the variables LANGFUSE_SECRET_KEY, LANGFUSE_PUBLIC_KEY, LANGFUSE_HOST must be defined in the .env file in the packages/server/ directory.

The name of the prompt must be the exact name saved in the repository. The version of the prompt will be added in the future.

You can override the name of the template and variables like this:

```python
import requests

API_URL = "api_url"


def query(payload):
    response = requests.post(API_URL, json=payload)
    return response.json()


output = query(
    {
        "question": "Hi, how are you?",
        "overrideConfig": {
            "template": "name_of_prompt",
            "promptValues": {
                "var1": "value1",
                "..."
            },
        },
    }
)
```

### CODE REVIEW

File: packages/components/nodes/prompts/PromptLangfuse/PromptLangfuse.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    constructor() {
+        this.label = 'LangFuse Prompt Template'
+        this.name = 'promptLangFuse'
+        this.version = 1.0
+        this.type = 'PromptTemplate'
+        this.icon = 'prompt.svg'
+        this.category = 'Prompts'
+        this.description = 'Fetch schema from LangFuse to represent a prompt for an LLM'
+        this.baseClasses = [...getBaseClasses(PromptTemplate)]
```
Comment (HenryHengZJ) [inline packages/components/nodes/prompts/PromptLangfuse/PromptLangfuse.ts:25]: Hey! thanks for the contribution!

I've just added this [PR](https://github.com/FlowiseAI/Flowise/pull/2902) that allow users to add nodes created by community.

Can you add this `this.author = 'your name'` to here, so as we can differentiate that this is a node created by community, and will be shown on UI as well

Comment (HenryHengZJ) [reply packages/components/nodes/prompts/PromptLangfuse/PromptLangfuse.ts:25]: an example guide: https://docs.flowiseai.com/contributing/building-node

Comment (cruz-lucas) [reply packages/components/nodes/prompts/PromptLangfuse/PromptLangfuse.ts:27]: Done!

**Review by HenryHengZJ — APPROVED** [review]
thanks @cruz-lucas !

**Pull request merged**